pkg/modprobe/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM linuxkit/alpine:87a0cd10449d72f374f950004467737dbf440630 AS mirror
+
+RUN mkdir -p /out/etc/apk && cp -r /etc/apk/* /out/etc/apk/
+RUN apk add --no-cache --initdb -p /out \
+    musl
+RUN rm -rf /out/etc/apk /out/lib/apk /out/var/cache
+
+# keep only /bin/busybox and /sbin/modprobe symlink from busybox package
+RUN mkdir /out/bin && \
+    mkdir /out/sbin && \
+    cp /bin/busybox /out/bin/busybox && \
+    cp -P /sbin/modprobe /out/sbin/modprobe
```

Comment:
I think the space savings are minimal so in this case I prefer to just keep the base `busybox` intact instead of trying to save a couple of symlinks and text files.

This `modprobe` package is mostly for convenience so the main part is the `org.mobyproject.config` below.